cms/src/components/languages/CreateOrEditLanguageModal.vue

```diff
@@ -5,6 +5,8 @@ import LInput from "@/components/forms/LInput.vue";
 import LButton from "@/components/button/LButton.vue";
```

Comment:
- Are we ensuring that controls are disabled for users without edit access to the given language?
- Are we ensuring that setting a language to Default is disabled for users without language edit access to all languages?

The same checks should also be implemented in the API.
The API should also remove the "default" field from other language documents if set on a given doc. (CMS checks are only best effort but cannot be relied on - the API does the final check and quality control)